app/index.html

```diff
@@ -367,6 +367,7 @@ <h4 class="modal-title" id="gcc-error-label" data-bind="html: errorWarningLabel"
         <script src="scripts/sys-runtime.js"></script>
         <script src="scripts/router.js"></script>
         <script src="scripts/main.js"></script>
+        <script src="scripts/editor/token_highlighter.js"></script>
```

Comment:
This is part of a larger refactoring process. This folder will be used by me and @anant-singh for all the subsequent modules that we will be adding to the editor. Also, this will help compile all the modules related to the editor at one place instead of mixing them with everything else as it is currently.